src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
```

Comment:
The parameter name 'arg' is usually written as 'args' to imply multiple arguments. It is a common convention in Java.